v03_pipeline/lib/reference_datasets/hgmd.py

```diff
@@ -0,0 +1,30 @@
+import hail as hl
+
+from v03_pipeline.lib.model import ReferenceGenome
+
+HGMD_CLASSES = [
```

Comment:
If these are only ever used in the config its probably better just to define them there